Build.fsx

```diff
@@ -1,15 +1,43 @@
 ﻿#r @"packages/FAKE.Core/tools/FakeLib.dll"
 
 open Fake
+open Fake.AssemblyInfoFile
 open Fake.Testing
 open System
+open System.Diagnostics;
+open System.Text.RegularExpressions
 
 let releaseFolder = "Release"
 let nunitToolsFolder = "Packages/NUnit.Runners.2.6.2/tools"
 let nuGetOutputFolder = "NuGetPackages"
 let solutionsToBuild = !! "Src/*.sln"
 let signKeyPath = FullName "Src/AutoFixture.snk"
 
+type GitVersion = { apiVersion:string; nugetVersion:string; }
+let getGitVersion = 
```

Comment:
I do like idea of switching to GitVersion tool, FAKE even have helpers for it, however I don't have experience with that tool. It would take some time for me to read the documentation.

Currently I'm focused on migration to .NET Core, so would like to have this PR merged as soon as possible.
Would it be OK for you if merge this PR as is and switch to `GitVersion` in `v4` branch later?